libs/deal-ticket/src/components/deal-ticket/deal-ticket.spec.tsx

```diff
@@ -402,7 +374,7 @@ describe('DealTicket', () => {
     render(generateJsx());
 
     act(() => {
-      screen.getByTestId(`order-type-${Schema.OrderType.TYPE_MARKET}`).click();
+      screen.getByTestId('order-type-Market').click();
```

Comment:
userEvent?

Reply:
This is exising test, just updating selector